gci/models.py

```diff
@@ -1 +1,70 @@
-# Create your models here.
+from django.db import models
+
+
+class Students(models.Model):
+    student_id = models.IntegerField(default=0, primary_key=True)
+    student_name = models.CharField(max_length=100, default='')
+    program_year = models.ForeignKey('ProgramYear', null=True)
+
+    def __str__(self):
+        return self.student_name
+
+class Organizations(models.Model):
+    org_id = models.IntegerField(default=0, primary_key=True)
+    org_name = models.CharField(max_length=100, default='')
+    org_logo = models.ImageField(upload_to='', blank=True, default='')
+    org_url = models.URLField(default='', blank=True)
+    org_summary = models.TextField(max_length=300, default='')
+    students = models.ManyToManyField(Students)
+
+    def __str__(self):
+        return self.org_name
+
+class Tasks(models.Model):
+    name = models.CharField(max_length=200, default='')
+    description = models.TextField(max_length=500, default='')
+    status = models.IntegerField(default=1)
+    max_instances = models.IntegerField(default=1)
+    mentors = models.ManyToManyField('Mentors')
+    tags = models.ManyToManyField('Tags')
+    is_beginner = models.BooleanField(default=False)
+    categories = models.ManyToManyField('Categories')
+    time_to_complete_in_days = models.IntegerField(default=3)
+    external_url = models.URLField()
+    private_metadata = models.CharField(max_length=80, default='')
```

Comment:
This is the only field which I dont know much about. I havent tested if it can be blank.  I do know it is never blank for our dataset.